### ISSUE TITLE
floating point instructions not working

### ISSUE THREAD

**Issue by SurajNatekar28** [hm-riscv/vscode-riscv-venus#126]
when i run a assembly code all instructions related to integer arithmatic are successfully executed but floating point arithmatic instructions are not executed
Are floating point instructions supported on venus?

**Comment by wallento** [comment]
No, unfortunately the F extension is not supported. It wouldn't be excessively hard though to add, if you are interested to contribute it.
